contribs/drt/src/main/java/org/matsim/contrib/drt/optimizer/DefaultDrtOptimizer.java

```diff
@@ -95,6 +97,7 @@ public void notifyMobsimBeforeSimStep(@SuppressWarnings("rawtypes") MobsimBefore
 			requestInserter.scheduleUnplannedRequests(unplannedRequests.getSchedulableRequests());
 		}
 
+		returnToDepot(drtCfg.returnToDepotEvaluationInterval, drtCfg.returnToDepotTimeout);
```

Comment:
Maybe change `returnToDepot()` to `relocateVehiclesToDepot()?  Or `planReturnsToDepot()` so it's clearer what the method does?

Reply:
Done